lib/src/lifecycle_module.dart

```diff
@@ -297,6 +303,13 @@ abstract class LifecycleModule {
     return completer.future;
   }
 
+  /// Aliased to [unload].
+  ///
+  /// Deprecated: Use the method [unload] instead.
+  @deprecated
+  @override
+  Future<Null> dispose() => unload();
```

Comment:
We decided to use `@deprecated` instead of `@protected` because it would "protect" us from consumers calling `dispose` from within their modules as well as from outside.